src/log-report.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+// @flow
+import { listen } from "@ledgerhq/logs";
+
+const logs = [];
+const logLimit = 10; // the number of latest log we want to conserve
```

Comment:
That value should be something like 1000
Similarly to desktop

Reply:
fixed